### PR TITLE
Added support for pinned applications

### PR DESCRIPTION
**Pull request by faffolao**
See also NethServer/nethserver-httpd#34

### CODE REVIEW

File: ui/public/i18n/language.json [hunk 1]
```diff
@@ -45,6 +45,8 @@
   "hints_suggested": "Please, review the following settings",
   "add_shortcut": "Add shortcut",
   "remove_shortcut": "Remove shortcut",
+  "add_pin": "Add pin",
```
Comment (gsanchietti) [inline ui/public/i18n/language.json]: I'd change the label to:
```
Add to home page
```

File: ui/public/i18n/language.json [hunk 2]
```diff
@@ -45,6 +45,8 @@
...
+  "remove_pin": "Remove pin",
```
Comment (gsanchietti) [inline ui/public/i18n/language.json]: I'd change the label to:
```
Remove from home page
```

File: ui/public/i18n/language.json (hunk 1 above)
Comment (faffolao) [reply ui/public/i18n/language.json]: updated

File: ui/public/i18n/language.json (hunk 2 above)
Comment (faffolao) [reply ui/public/i18n/language.json]: updated

**Pull request closed without merging**